compiler/rustc_parse/src/validate_attr.rs

```diff
@@ -2,8 +2,12 @@
 
 use crate::parse_in;
 
+use rustc_ast::token::{Token, TokenKind};
 use rustc_ast::tokenstream::{DelimSpan, TokenTree};
-use rustc_ast::{self as ast, Attribute, MacArgs, MacDelimiter, MetaItem, MetaItemKind};
+use rustc_ast::{
+    self as ast, Attribute, MacArgs, MacArgsEqKind, MacDelimiter, MetaItem, MetaItemKind,
+};
```

Comment:
Nit: could you avoid multi-line imports?

Reply:
Renaming `MacArgsEqKind` as `MacArgsEq` was enough to get it back on a single line :)